test/clj/game_test/cards/resources.clj

```diff
@@ -803,6 +803,22 @@
       (prompt-choice :runner "Card from hand")
       (prompt-choice :runner "Steal"))))
 
+(deftest gang-sign-correct-prompts
+         ;; Gang Sign - accessing from HQ, not including root. Issue #2113.
+   (do-game
+     (new-game (default-corp [(qty "Hostile Takeover" 1) (qty "Snare!" 1)])
+               (default-runner [(qty "Gang Sign" 1)]))
```

Comment:
Since I found out that we can replace `(qty X 1)` with just `X` I prefer that for single copies of cards.